tests/kafkatest/services/kafka/kafka.py

```diff
@@ -460,38 +460,6 @@ def node_id_as_isolated_controller(self, node):
         """
         return self.idx(node) + config_property.FIRST_CONTROLLER_ID - 1
 
-    def reconfigure_zk_for_migration(self, kraft_quorum):
-        self.configured_for_zk_migration = True
-        self.controller_quorum = kraft_quorum
-
-        # Set the migration properties
-        self.server_prop_overrides.extend([
-            ["zookeeper.metadata.migration.enable", "true"],
-            ["controller.quorum.voters", kraft_quorum.controller_quorum_voters],
-            ["controller.listener.names", kraft_quorum.controller_listener_names]
-        ])
-
-        # Add a port mapping for the controller listener.
-        # This is not added to "advertised.listeners" because of configured_for_zk_migration=True
-        self.port_mappings[kraft_quorum.controller_listener_names] = kraft_quorum.port_mappings.get(kraft_quorum.controller_listener_names)
-
-    def reconfigure_zk_as_kraft(self, kraft_quorum):
-        self.configured_for_zk_migration = True
```

Comment:
Please remove this variable completely (along with its other usages)

Reply:
Thanks for the reminder! Also, combine the if-else statement in the function `controller_listener_name_list` since the variable `configured_for_zk_migration` will never be set to true.